src/rqt_embed_window/embed_window_widget.py

```diff
@@ -31,7 +29,7 @@ def get_window_id_by_window_name(window_name):
             # Avoiding dealing with unicode...
             if str(this_window_name) == str(window_name):
                 return int(fields[0], 16)
-    return None
+    return
```

Comment:
I believe that being explicit is better than being implicit, I'd prefer to leave the explicit return of `None` than to make it implicit.